library/core-components/components/typography/typography.stories.tsx

```diff
@@ -24,95 +25,45 @@ const meta: Meta<typeof Typography> = {
     align: {
       defaultValue: 'left',
     },
-    color: {
-      defaultValue: '--color-text-on-base-primary',
-      options: [
-        '--color-text-on-base-primary',
-        '--color-text-on-base-secondary',
-        '--color-text-on-base-accent',
-        '--color-text-on-base-disabled',
-        '--color-text-inverse-primary',
-        '--color-text-inverse-secondary',
-        '--color-text-inverse-accent',
-        '--color-text-on-subtle-primary',
-        '--color-text-on-subtle-secondary',
-        '--color-text-on-subtle-accent',
-        '--color-text-on-subtle-disabled',
-        '--color-text-on-accent-primary',
-        '--color-text-on-accent-secondary',
-        '--color-text-on-accent-disabled',
-        '--color-text-primary-action-default',
-        '--color-text-primary-action-hover',
-        '--color-text-primary-action-active',
-        '--color-text-primary-action-focus',
-        '--color-text-primary-action-disabled',
-        '--color-text-secondary-action-default',
-        '--color-text-secondary-action-hover',
-        '--color-text-secondary-action-active',
-        '--color-text-secondary-action-focus',
-        '--color-text-secondary-action-disabled',
-        '--color-text-tertiary-action-default',
-        '--color-text-tertiary-action-hover',
-        '--color-text-tertiary-action-active',
-        '--color-text-tertiary-action-focus',
-        '--color-text-tertiary-action-disabled',
-      ],
-      control: 'select',
+    fill: {
+      options: flattenObject(radiusTokens.component.color),
     },
     font: {
-      defaultValue: '--typography-body-md',
-      options: [
-        '--typography-heading-xxl',
-        '--typography-heading-xl',
-        '--typography-heading-lg',
-        '--typography-heading-md',
-        '--typography-heading-sm',
-        '--typography-heading-xs',
-        '--typography-body-xl',
-        '--typography-body-md',
-        '--typography-body-sm',
-        '--typography-link-sm',
-        '--typography-btn-lg',
-        '--typography-btn-md',
-        '--typography-btn-sm',
-      ],
-      control: 'select',
+      options: flattenObject({
+        core: radiusTokens.core.typography,
+        semantic: radiusTokens.semantic.typography,
+        component: radiusTokens.component.typography,
```

Comment:
We still need to align on which tokens to allow in these components. If it really is just storybook that benefits from non-component tokens then it may make sense to restrict these types and just ignore the type errors in storybook where necessary.